lib/hooks/useStarSearchFeedback.ts

```diff
@@ -0,0 +1,23 @@
+import { usePostHog } from "posthog-js/react";
+
+export interface StarSearchFeedbackAnalytic {
+  feedback: "positive" | "negative";
+  promptContent: string[];
+  promptResponse: string[];
+}
+
+export const useStarSearchFeedback = () => {
+  const posthog = usePostHog();
+
+  const feedback = ({ feedback, promptContent, promptResponse }: StarSearchFeedbackAnalytic) => {
+    posthog.capture("star_search_feedback", {
+      feedback,
+      promptContent,
+      promptResponse,
```

Comment:
Perfect: is this data persisted in Posthog long term?

---

I've also wondered about just storing the queries / responses people are getting when they use it. Might be some valuable data in there regardless of if they're giving us feedback or not

Reply:
I'm not sure how long this data is persisted in Posthog.I can dig into that or maybe @isabensusan knows.

Reply:
We can also investigate a more full fledged solution via https://github.com/open-sauced/api/issues/751 if we want to start doing a chat "history" or "memory" of what was said before, we'll need this in the API anyways.

Reply:
Story history in app and across workspaces is something came up in a demo today.